apps/paper/pages/programmable-intents/operators.mdx

```diff
@@ -1 +1,52 @@
+2. It is imperative that at least a third of guardians are actively managing their full node setups across all k chains, thus guaranteeing the availability and reliability of the network infrastructure.
+3. Upon satisfying the aforementioned conditions, guardians are afforded the flexibility to select additional chains of their preference for further engagement, allowing for adaptive and strategic distribution of network resources.
+
+### Guardian Consensus
+
+If a majority vote of > 2/3 of the guardian sign the same message, it can then be considered as valid on the LL network. Once signed, the solver will initiate the following transactions outlined by the intent of the user.
+
+The Guardian Network is the core component of the Light ecosystem. This network is pivotalfor Light Layer functionality, which aims to be a top interoperability platform by emphasizing five key features: 
+- Decentralization: ensuring network control is widely distributed
+- Modularity, allowing for independent design and updates of various ecosystem parts
+- Chain Agnosticism, supporting a wide range of blockchain platforms without relying on any single one
+- Scalability, capable of managing large values and transaction volumes
+- Upgradeability, facilitating updates in response to the evolving decentralized computing landscape.
+
+## Auctioneers
+
+In the realm of cross-chain transactions, the LL introduces an innovative and efficient routing mechanism through the integration of an auction system, complementing the traditional bridge-based routing solutions. Users are empowered to define a maximum fee threshold for the transaction services rendered by auctioneers. This setup initiates a competitive bidding process, known as an English Auction, wherein auctioneers vie to offer the most cost-effective and efficient transaction routing options within the specified fee limit. The auction is strategically timed to commence and conclude within the duration of a single blockchain block, thus ensuring that transaction processing is not only competitive in terms of pricing but also expedited for user convenience.
+
+Should the final transaction cost fall below the user-specified maximum fee, the LL guarantees the reimbursement of the residual amount to the user. This approach not only fosters a competitive environment for transaction routing but also ensures transparency and fairness in transaction costs, ultimately benefiting the end users by providing them with cost-effective and timely transaction services. Through these meticulously designed operational roles and mechanisms, the LL exemplifies a forward-thinking approach to blockchain infrastructure management, emphasizing security, efficiency, and user-centricity.
+
+### Cross-Chain Trilemma
+
+![Cross-Chain Trilemma](/cross-chain-trilemma.png)
```

Comment:
Probably, we should cite + get approval from frontier.tech's Ankit and Stephane

Reply:
Yep. Going to cite the entire trilemma section

Reply:
Thank you so much!